### DISCUSSION TITLE
Confusion of input parameters in manual

### DISCUSSION THREAD

**Discussion by scott-5** [cp2k/cp2k#2446] [Ideas]
In my opinion, setting `FORCE_EVAL/DFT/SCF/DIAGONALIZATION/ALGORITHM` to OT is equivalent to set `FORCE_EVAL/DFT/SCF/OT` to .TRUE.. And there also is the same subsection `OT` in `FORCE_EVAL/DFT/SCF/DIAGONALIZATION`.
So can OT be considered as one of algorithm to handling diagonalization? If so, then why `FORCE_EVAL/DFT/SCF/OT` is separated from `FORCE_EVAL/DFT/SCF/DIAGONALIZATION`? This can easily lead to lose hierarchy. That is not good for memory, even conceptual confusion.
Why not remove `FORCE_EVAL/DFT/SCF/OT` to make it more standard?

**Comment by mkrack** [comment]
The `OT` section appears twice, because `OT` is used in different contexts. `SCF/OT` performs just a single `OT` step per SCF iteration step (i.e. no diagonalization is done), whereas `SCF/DIAGONALIZATION/OT` defines the input parameters for `OT`, if it is selected as the `SCF/DIAGONALIZATION/ALGORITHM` to perform a diagonalization which involves a full `OT` iteration cycle in each SCF iteration step.

**Reply by scott-5** [reply]
Thanks for your explanation!!! It makes me more clear about `OT`.
